superset/views/core.py

```diff
@@ -815,7 +815,7 @@ def explore(
             try:
                 datasource = DatasourceDAO.get_datasource(
                     db.session,
-                    DatasourceType(cast(str, datasource_type)),
+                    DatasourceType("table"),
```

Comment:
@hughhhh @pkdotson I see that this endpoint is deprecated but why are we only using table here?